plugins/table.md

```diff
@@ -80,6 +80,26 @@ tinymce.init({
 });
 ```
 
+### `table_use_colgroups`
```

Comment:
Just realised this is missing the `{{ site.requires_5_5v }}` note sorry Hannes. eg:

```
### `table_use_colgroups`

{{ site.requires_5_5v }}

This option allows you to set whether `colgroup`s should be added to new tables created by the table plugin. Tables inserted by other means, or already existing tables, will not be changed.
```